ironfish/src/network/utils/serializers.ts

```diff
@@ -51,23 +52,23 @@ export function readBlockHeader(reader: bufio.BufferReader): SerializedBlockHead
   const minersFee = (-BigIntUtils.fromBytesLE(reader.readBytes(8))).toString()
   const graffiti = GraffitiSerdeInstance.serialize(reader.readBytes(32))
```

Comment:
You can skip the GraffitiSerdeInstance entirely here and just pass the bytes through from the reader.

Reply:
Changed